go.mod

```diff
@@ -1,21 +1,21 @@
 module github.com/ramendr/ramen
 
-go 1.22.5
+go 1.22.7
 
-toolchain go1.22.7
+toolchain go1.23.3
```

Comment:
@Madhu-1 do you know why these changed when I updated csi-addons?

Reply:
@raghavendra-talur think this is caused by using go 1.23.3 when running `go mod tidy`.

> ok, so googling leads me to go env GOTOOLCHAIN
> I think we need a makefile/env variable that sets it for the project
> the default is local and therefore it picks from the developer's system

We have go 1.22.9 in downstream (on the build machines) so we can update both go version and toolchain to go 1.22.9 now.